### PR TITLE
Fixes bug with associating user with idea

### PR DESCRIPTION
**Pull request by jmichaelbrown8**
#73 

This fixes the bug where an idea doesn't save the user's id and the idea.handlebars always displaying the logged in user (rather than the idea creator).

### CODE REVIEW

**Review by kimberlym4488 — APPROVED** [review]

**Review by ac524 — APPROVED** [review]
Looks good to me :)

File: models/index.js [hunk 1]
```diff
@@ -73,22 +78,23 @@ Idea.hasMany(Comment, {
...
 });
 
-// Idea.hasMany(IdeaUpvote, {
-//   foreignKey: 'idea_id',
-// });
+Idea.hasMany(IdeaUpvote, {
+  through: IdeaUpvote,
+  as: 'upvoter',
+  foreignKey: 'idea_id',
+});
```
Comment (ac524) [inline models/index.js:90]: Hey @jmichaelbrown8, in merging changes from the idea upvote branch, we now have an alias conflict over "upvoter", We'll need a new alias name for this.

**Review by ac524 — CHANGES_REQUESTED** [review]

File: models/index.js (hunk 1 above)
Comment (ac524) [reply models/index.js:90]: Or maybe I did the merge wrong and the hasMany isn't needed? Not sure in this case without looking through uses more.

**Review by ac524 — APPROVED** [review]
Fixed alias issue, so this is good to go.

**Pull request merged**